RELEASE.MD

```diff
@@ -0,0 +1,93 @@
+# Releasing a new version of FAIR Connect
+
+## 1. Start the release workflow
+
+1. Go to the FAIR Connect [repository](https://github.com/fairpm/fair-plugin/actions).
+2. Click the **Actions** tab.
+3. In the *Actions* workflow list, select **Bump version for release**.
+
+## 2. Configure and run the workflow
+
+1. Click the **Run workflow** button. A workflow input panel opens.
+2. Complete the following fields:
+
+- **Use workflow from:** Select **Branch: main** (default).
+- **New version being released:** Enter the release version number (e.g., `1.0.0`).
+  FAIR Connect uses semantic versioning (`MAJOR.MINOR.PATCH`).
+
+3. Click the **Run workflow** button to start the release process.
+
+4. Refresh the page to view workflow progress.
+5. Click the running workflow to open the logs.
+6. When the workflow finishes, it creates a pull request containing the version-bump changes.
+
+## 3. Review and merge the version bump PR
+
+1. Go to the **Pull requests** tab.
+2. Open the version-bump pull request created by the release workflow.
+3. Review the changes:
+
+    - Update of the version number in `plugin.php`
+    - Update the `VERSION` constant in `plugin.php`
+
+4. Confirm that the changes are correct.
+5. After review and approval, merge the PR to the `main` branch.
+6. Go to the **Actions** tab to verify that workflow processing is complete.
+7. Continue to the next step once all workflows finish.
+
+## 4. Review and merge the Generate POT PR
+
+1. Go to the **Pull requests** tab.
+2. Open the most recent **Generate POT** pull request.
+3. Review the changes:
+
+    - Update `POT-creation-date` in `languages/fair.pot`
+    - Update package DID in `languages/fair.pot`
+    - Update package URL in `languages/fair.pot`
```

Comment:
These just happen to be the changes (specifically, line number changes) in the current POT PR.

Suggest that the list of changes to review are:
```
- Update to `Project-Id-Version` in `languages/fair.pot`
- Update to `POT-Creation-Date` in `languages/fair.pot`
```